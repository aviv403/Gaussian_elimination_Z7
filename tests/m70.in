2 4 1 4 0 4 5 5 2 2 5
5 5 6 2 3 3 1 2 3 6 4
3 0 5 0 2 5 1 2 2 6 3
2 0 6 3 1 2 5 4 3 0 1
