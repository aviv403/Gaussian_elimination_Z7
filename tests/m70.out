1 0 0 0 5 3 5 0 4 5 0
0 1 0 0 4 2 1 4 6 6 5
0 0 1 0 3 2 0 6 5 1 2
0 0 0 1 5 4 3 1 0 4 1
